lib/language_pack/installers/heroku_ruby_installer.rb

```diff
@@ -22,5 +22,20 @@ def fetch_unpack(ruby_version, install_dir, build = false)
       @fetcher.fetch_untar(file)
     end
   end
+
+  def setup_binstubs(*)
```

Comment:
Is this the right file to add this logic? We use this to installer to download the version of Ruby that the buildpack bootstraps with. 

It gets called here https://github.com/heroku/heroku-buildpack-ruby/blob/a46dcce3f35881dea142d609e7e42bf30ebc7549/bin/support/download_ruby#L12 

and that gets called from this script https://github.com/heroku/heroku-buildpack-ruby/blob/a46dcce3f35881dea142d609e7e42bf30ebc7549/bin/support/bash_functions.sh#L35 which is executed in a few places

https://github.com/heroku/heroku-buildpack-ruby/search?utf8=%E2%9C%93&q=heroku_buildpack_ruby_install_ruby&type=

I don't think we're inside of the right directory either.